<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerMarkupsPlaceWidget</class>
 <widget class="qSlicerWidget" name="qSlicerMarkupsPlaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Place markups</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="PlaceButton">
     <property name="toolTip">
      <string>Place a markup point</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources/qSlicerMarkupsModule.qrc">
       <normaloff>:/Icons/MarkupsMouseModePlace.png</normaloff>:/Icons/MarkupsMouseModePlace.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="DeleteLastButton">
     <property name="toolTip">
      <string>Delete last added markup point</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources/qSlicerMarkupsModule.qrc">
       <normaloff>:/Icons/MarkupsDelete.png</normaloff>:/Icons/MarkupsDelete.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
  <action name="ActionPersistent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Place multiple markups</string>
   </property>
   <property name="toolTip">
    <string>Place multiple markups</string>
   </property>
  </action>
  <action name="ActionDeleteAll">
   <property name="icon">
    <iconset resource="../../../Resources/qSlicerMarkupsModule.qrc">
     <normaloff>:/Icons/MarkupsDeleteAllRows.png</normaloff>:/Icons/MarkupsDeleteAllRows.png</iconset>
   </property>
   <property name="text">
    <string>Delete all markups</string>
   </property>
   <property name="toolTip">
    <string>Delete all markups in the list</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc"/>
  <include location="../../../Resources/qSlicerMarkupsModule.qrc"/>
 </resources>
 <connections/>
</ui>
